modules/op-release-notes-1-19.adoc

```diff
@@ -471,6 +471,8 @@ The existing job-based pruner must be disabled before enabling the event-based p
 
 * With this release, the `opc results list` command is replaced with the `opc results result list` command.
 
+* With this update, the `disable-affinity-assistant` flag is removed from the {pipelines-title} Operator. This flag is deprecated in {pipelines-title} v1.13 and does not have any effect in {pipelines-title} v1.19. The `disable-affinity-assistant flag is still available in the `TektonConfig` custom resource (CR) for backward compatibility, but it does not impact the behavior of the {pipelines-title} Operator. To maintain the same behavior, set the `coschedule` feature flag to `disabled`.
```

Comment:
We don't have an example to set this coschedule flag in the Tekton config 
Correct me if I am wrong

Reply:
Fixed.